### ISSUE TITLE
Multiple calls to Markdown.ToHtml with same pipeline parameter throws an exception

### ISSUE THREAD

**Issue by pekkah** [lunet-io/markdig#4]
When calling Render method of following class more than once `Markdown.ToHtml` throws an exception (at the bottom). It seems to be related to the `_pipeline` parameter. If I remove it from the parameters multiple calls work as expected.

Class:
```
 public class MarkdigRenderer : IMarkdownRenderer
    {
        private readonly MarkdownPipeline _pipeline;

        public MarkdigRenderer()
        {
            _pipeline = new MarkdownPipeline().UseAllExtensions();
            _pipeline.DebugLog = new StringWriter();
        }

        public string Render(string markdown)
        {
            var html =  Markdown.ToHtml(markdown, _pipeline);

            return html;
        }
    }
```

Exception
```
System.InvalidOperationException: The character `:` is already used by another emphasis descriptor
   at Markdig.Parsers.Inlines.EmphasisInlineParser.Initialize(InlineProcessor processor)
   at Markdig.Parsers.ParserList`2.Initialize(TState initState)
   at Markdig.Parsers.InlineParserList.Initialize(InlineProcessor initState)
   at Markdig.Parsers.InlineProcessor..ctor(StringBuilderCache stringBuilders, MarkdownDocument document, InlineParserList parsers)
   at Markdig.Parsers.MarkdownParser..ctor(TextReader reader, MarkdownPipeline pipeline)
   at Markdig.Parsers.MarkdownParser.Parse(TextReader reader, MarkdownPipeline pipeline)
   at Markdig.Markdown.ToHtml(TextReader reader, TextWriter writer, MarkdownPipeline pipeline)
   at Markdig.Markdown.ToHtml(TextReader reader, MarkdownPipeline pipeline)
   at Markdig.Markdown.ToHtml(String markdown, MarkdownPipeline pipeline)
````

**Comment by xoofx** [comment]
Thanks this should be fixed by commit 09beb2f (nuget version: 0.1.1)

**Comment by xoofx** [comment]
Note that latest version (0.2.0+) is introducing a `MarkdownPipelineBuilder` instead to disallow mutable pipeline (related to issue #5).

Replace

```
var pipeline = new MarkdownPipeline().UseAllExtensions()
```

by:

```
var pipeline = new MarkdownPipelineBuilder().UseAllExtensions().Build();
```
